wiki/Beatmap_Editor/fr.md

```diff
@@ -36,7 +38,6 @@ L'éditeur de beatmap inclut également :
 - [AiMod](AiMod) : Un système automatisé qui signale les problèmes dans une beatmap.
 - [Beat snap divisor](Beat_Snap_Divisor) : Détermine comment les objets sont fixés à la timeline.
 - [Distance snap](Distance_Snap) : Détermine la distance entre 2 objets consécutifs en prenant en compte le rythme de la musique.
-- [Kiai time](Kiai_Time) : Des sections spécifiques du timing qui aident à mettre en valeur une partie de la beatmap.
```

Comment:
please don't remove it, fix the link instead

Reply:
It will be done